src/Card/Card.Content.js

```diff
@@ -0,0 +1,62 @@
+import { dcnb } from "cnbuilder";
+import React from "react";
+import PropTypes from "prop-types";
+import { FlexBox } from "../FlexBox/FlexBox";
+import { cardAlignOptions } from "./Card.levers";
+
+/**
+ * Content Component.
+ * @param {*} param0
+ * @returns
+ */
+
+export const Content = ({ children, className, align, ...props }) => {
+  // Content alignment including image and CTA, default is left-align
+  // This setting overrides the alignment option in the nested CTA
+  let bodyAlign = "su-items-start";
+  let contentClass = "";
+
+  if (align === "center") {
+    contentClass = dcnb(contentClass, "su-text-center");
```

Comment:
```suggestion
    contentClass = "su-text-center";
```

contentClass at this point cannot be anything other than an empty string. No sense in running `dcnb` on it.

Reply:
I actually am thinking about moving `align` to the parent Card.Root, since sometimes alignment needs to include the image (eg, if they swap out the full width card image with a small thumbnail like we did on SAA.

Reply:
Hmmm.....nevermind. Until we add the children variant plugin to the Decanter 7 repo, it's simpler to leave the align prop in the Card.Content subcomponent. Maybe later 🤔